Sebastian-Sole/handicappin
# 💄 feat: update add-round UI

## What was done
- Updated the UI for the Add Round form. The form now has two cards: one for course info and one for holes played
- Added a skeleton component for the form
- Removed the formSkeleton component
- Added support for loading states when a user is saving a round
- Removed the "Populate DB" testing button
- Updated the "todays date" calendar background color for more of a contrast to the selected background color

## How to test
- Login
- Visit /rounds/add
- See the updated round form and check responsivity
- Fill in the form and submit
- See that inputs are disabled

## Future work
- None
